src/components/HomePage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function HomePage() {
+  return (
+    <div>
+      Welcome to homepage
+    </div>
+  );
```

Comment:
- [ ] you should have an array of objects then you call them props in the books.js.

![ppp](https://user-images.githubusercontent.com/61562626/146035380-154dcc2d-29bd-4eec-893a-315c29372e9a.jpeg)

Reply:
Thanks, @dasileker, recommendation implemented.